src/Symfony/Component/Messenger/Tests/Fixtures/App/Kernel.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Symfony\Component\Messenger\Tests\Fixtures\App;
+
+use Symfony\Bundle\FrameworkBundle\FrameworkBundle;
+use Symfony\Bundle\FrameworkBundle\Kernel\MicroKernelTrait;
+use Symfony\Component\Config\Loader\LoaderInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\HttpKernel\Kernel as SymfonyKernel;
+use Symfony\Component\Messenger\MessageBus;
+use Symfony\Component\Messenger\MessageBusInterface;
+
+class Kernel extends SymfonyKernel
+{
+    use MicroKernelTrait;
+
+    public function registerBundles(): iterable
+    {
+        yield new FrameworkBundle();
+    }
+
+    public function registerContainerConfiguration(LoaderInterface $loader): void
+    {
+        $loader->load(function (ContainerBuilder $container) use ($loader) {
+            $container->loadFromExtension('framework', [
+                'router' => [
+                    'resource' => 'kernel::loadRoutes',
+                    'type' => 'service',
+                ],
+            ]);
+            $container
+                ->register('message.bus', MessageBus::class);
+            $container->setAlias(MessageBusInterface::class, 'message.bus')->setPublic(true);
+        });
+    }
+
+    public function getCacheDir(): string
+    {
+        return sys_get_temp_dir().'/'. \Symfony\Component\HttpKernel\Kernel::VERSION.'/EmptyAppKernel/cache/'.$this->environment;
+    }
+
+    public function getLogDir(): string
+    {
+        return sys_get_temp_dir().'/'.Kernel::VERSION.'/EmptyAppKernel/logs';
```

Comment:
```suggestion
        return sys_get_temp_dir().DIRECTORY_SEPARATOR.Kernel::VERSION.DIRECTORY_SEPARATOR.'EmptyAppKernel'.DIRECTORY_SEPARATOR.'logs';
```

and others in this file?